src/riak_kv_compile_tab.erl

```diff
@@ -71,35 +100,75 @@ is_compiling(BucketType) ->
         compiling_state() | notfound.
 get_state(BucketType) when is_binary(BucketType) ->
     case dets:lookup(?TABLE, BucketType) of
-        [{_,_,_,State}] ->
+        [{_,_,_,_,State}] ->
```

Comment:
Guess that means we'd have to do it everywhere it could crash, which is probably overkill.